AirCasting/AuthViews/UserAuthenticationSession.swift

```diff
@@ -136,3 +148,38 @@ final class DefaultLogoutController: LogoutController {
         assertionFailure("[LOGOUT] Failed to log out \(error)")
     }
 }
+
+protocol DeleteAccountController {
+    func deleteAccount(completion: @escaping (Result<String, Error>) -> Void) throws
+}
+
+final class DefaultDeleteAccountController: DeleteAccountController {
```

Comment:
Should we keep all those classes in one file? Maybe it would be better to keep each in a separate file? Unless @pawelgil suggested otherwise, but then I'm curious what is the reason for keeping them in a single file

Reply:
Yes, I agree 100%